ix-dev/stable/nextcloud/app.yaml

```
@@ -56,6 +58,11 @@ run_as_context:
   group_name: root
   uid: 0
   user_name: root
+- description: Imaginary runs as user 65534.
+  gid: 65534
+  group_name:
+  uid: 65534
+  user_name:
```

```suggestion
- description: Imaginary runs as non-root user.
  gid: 65534
  group_name: imaginary
  uid: 65534
  user_name: imaginary
```

Those can't be empty, lets set them to something relevant atleast

---

Setting the user to 568 worked just fine.